source/creating_content/01_creating_content_page/managing_page_security.rst

```diff
@@ -0,0 +1,117 @@
+
+- *Browse page* permission:
+
+  - For a private page, the *Browse page* permission gives access to that page, hence the importance of adding users to the security management section, otherwise nobody (except users with the general domain-level *Browse all pages*) can access the page and its content.
+  - For a public page, the *Browse page* permission doesn't change much to the access security of that page because it is already accessible by anybody who has access to the domain.
+
+- *Edit page* permission: this permission is relevant for both private and public pages and gives edition rights on the page to any user/group of users added to the security management section of the page and granted with this *Edit page* permission.
+
+Adding and deleting users or groups of users from a page
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Adding users or groups to a page
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+1. Go to Security > User permissions or Security > Group permissions, depending on whether a user or a group will be added.
+2. Click the **Add user/group permissions** button at the bottom of the table.
+3. Select one or more users/groups of users from the list. A search bar is also available to search a user/group by name.
+4. (optional) If all added users/groups are going to be granted the *Edit page* permission as well as the default *Browse page* permission, it is possible to grant that permission now. Otherwise permissions can be granted afterward. To already grant the *Edit page* permission, at the top of the "Add user/group permissions" window, in the "Grant these permissions" section:
+
+  a. Click the **Add permission** button.
+  b. Click the *Edit page* permission button.
+
+5. Click the **Add user/group permissions** button at the bottom of the Add user/group permissions window.
+6. Click the **Save** button in the top right corner.
+
+Deleting users or groups from a page
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+1. Go to Security > User permissions or Security > Group permissions, depending on whether a user or a group will be deleted.
+2. In the table, the first column indicates the name of the user or the group of users. The second column indicates the corresponding permission(s). The third column displays the |trash-button| button. Click that |trash-button| button to delete a chosen user/group of users.
+3. Click the **Save** button in the top right corner.
+
+Managing page security permissions for users and groups of users
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Managing page security permissions can mean 2 things:
+
+- granting a user or a group of users a new page permission,
+- or, deleting a page permission for a user or a group of users.
+
+.. admonition:: Note
+   :class: note
+
+   Permissions can be granted beforehand, when adding the users and groups of users to the page (see `Adding users or groups to a page`_ for more information).
```

Comment:
I don't understand this note, I think it means that sometimes there already are users when you go manage a page security, but it seems obvious, no?

Maybe the note could be "if permissions have already been granted to users during their creation, this step is optional"?